content/PIV/Guides/SSH_with_PIV_and_PKCS11.adoc

```diff
@@ -1,76 +1,59 @@
-== Using PIV for SSH through PKCS11
-This is a step-by-step for how to get a YubiKey with PIV to work for
-public-key authentication with OpenSSH through PKCS11.
-Primarily on a OS X or Linux system.
+== Using PIV for SSH through PKCS #11
+This is a step-by-step guide on setting up a YubiKey with PIV to work for public-key authentication with OpenSSH through PKCS #11. These instructions apply primarily to OS X and Linux systems.
 
 === Prerequisites
 * a YubiKey with the PIV application loaded
-* the yubico-piv-tool software
+* the yubico-piv-tool software (download from link:../yubico-piv-tool/Releases/)
 * the OpenSC software
 * OpenSSH
-** If you are using OSX El Capitan (10.11) or earlier, for ssh-agent to work a newer OpenSSH than is delivered with the system; macOS Sierra (10.12) contains a compatible version
+* For OS X, iOS 10.13 and later are supported
 
 [NOTE]
-The following example assume that you have not yet changed the management key. If you have changed the management key, add `--key` to the `yubico-piv-tool -a import-certificate` command below.
+RSA 4096-bit keys are not currently supported due to a limitation in the PIV spec: https://github.com/Yubico/yubico-piv-tool/issues/58
+
+[NOTE]
+The examples given in the following steps assume that you have not yet changed the management key. If you have changed the management key, add `--key` to the `yubico-piv-tool -a import-certificate` command below.
```

Comment:
Maybe add a sentence saying that we strongly recommend changing the management key, or that we strongly discourage users from keeping the default management key

Reply:
Done. And when the recommendation is this strong, emphasizing it as you did seems a good idea: 
[NOTE]
We strongly recommend changing the management key; keeping the default management key is explicitly discouraged. The examples given in the following steps assume that you have not yet changed the management key. If you have changed the management key, add `--key` to the `yubico-piv-tool -a import-certificate` command below.